packages/auth/src/api/index.ts

```diff
@@ -181,7 +181,12 @@ export async function _performFetchWithErrorHandling<V>(
     if (e instanceof FirebaseError) {
       throw e;
     }
-    _fail(auth, AuthErrorCode.NETWORK_REQUEST_FAILED);
+    if (e instanceof Error) {
+      _fail(auth, AuthErrorCode.INTERNAL_ERROR, e);
```

Comment:
Can we instead do:

```
_fail(auth, AuthErrorCode.INTERNAL_ERROR, {"message": String(e)});
```
without an if branch?

Reply:
Done.